src/covid_shared/cli_tools.py

```diff
@@ -58,6 +58,10 @@ def configure_logging_to_files(output_path: Path) -> None:
     click.option('-v', 'verbose',
                  count=True,
                  help='Configure logging verbosity.'),
+    click.option('-e, --email-alert',
+                 type=bool,
+                 default=False,
+                 help='Send email when final job is done')
```

Comment:
Doesn't this decorator need `is_flag=True`?

You don't provide an option to specify the recipient(s). Is the plan to hard-code just the calling user? That will probably work fine, but I'd suggest adding that information to the `--email-alert` help text.

Reply:
You're completely right. Fixed that and updated the help text.